demo/config.yaml

```diff
@@ -23,28 +23,4 @@ lcr-modules:
 
     liftover:
         inputs:
-            sample_seg: "data/{sample_id}_subclones.hg38.igv.seg"
-        scratch_subdirectories: "scratch/"
-        options:
-            seghg38tobedhg38: "{MODSDIR}/src/convert_for_liftover.py"
-            bedhg38tobedhg19: "{MODSDIR}/src/chains/hg38ToHg19.over.chain"
-            bedhg19toseghg19: "{MODSDIR}/src/convert_for_liftover.py"
-        conda_envs:
-            liftover-366: "{MODSDIR}/envs/liftover-366.yaml"
-        threads:
-            seghg38tobedhg38: 12
-            bedhg38tobedhg19: 12
-            bedhg19toseghg19: 12
-        mem_mb:
-            seghg38tobedhg38: 100000
-            bedhg38tobedhg19: 100000
-            bedhg19toseghg19: 100000
-        pairing_config:
-            genome:
-                run_paired_tumours: True
-                run_unpaired_tumours_with: null
-                run_paired_tumours_as_unpaired: False
-            capture:
-                run_paired_tumours: True
-                run_unpaired_tumours_with: null
-                run_paired_tumours_as_unpaired: False
+            sample_seg: "data/{tumour_sample_id}_subclones.hg38.igv.seg"
```

Comment:
This is the demo config. You shouldn't be committing edits to this file

Reply:
Isn't it helpful to demo how to use new modules as they're introduced?

Reply:
Sorry, my mistake. I see that you have retained the part of the config for the demo. This file needs a newline added at the end.

Reply:
Also, I see now that the new modules, such as sequenza and strelka were added to this file. I will push the updated version of this file that includes these modules, and a new line at the end